README.md

```diff
@@ -14,62 +14,61 @@ This version supports ROS2 Dashing, Eloquent, Foxy, Galactic and Rolling.
+      - In that case treat yourself as a developer. Make sure you follow the instructions to also install librealsense2-dev and librealsense2-dkms packages.  
 
-    Install librealsense2 debian package:
-    * Jetson users - use the [Jetson Installation Guide](https://github.com/IntelRealSense/librealsense/blob/master/doc/installation_jetson.md)
-    * Otherwise, install from [Linux Debian Installation Guide](https://github.com/IntelRealSense/librealsense/blob/master/doc/distribution_linux.md#installing-the-packages)
-      - In that case treat yourself as a developer. Make sure you follow the instructions to also install librealsense2-dev and librealsense2-dkms packages.
+- #### Option 2: Build from source
+  - Download the latest [Intel&reg; RealSense&trade; SDK 2.0](https://github.com/IntelRealSense/librealsense/releases/tag/v2.50.0)
+  - Follow the instructions under [Linux Installation](https://github.com/IntelRealSense/librealsense/blob/master/doc/installation.md)
 
-   #### OR
-   - #### Build from sources by downloading the latest [Intel&reg; RealSense&trade; SDK 2.0](https://github.com/IntelRealSense/librealsense/releases/tag/v2.50.0) and follow the instructions under [Linux Installation](https://github.com/IntelRealSense/librealsense/blob/master/doc/installation.md)
 
-
-   ### Step 3: Install Intel&reg; RealSense&trade; ROS2 wrapper from Sources
+### Step 3: Install Intel&reg; RealSense&trade; ROS2 wrapper from sources
    - Create a ROS2 workspace
-   ```bash
-   mkdir -p ~/ros2_ws/src
-   cd ~/ros2_ws/src/
-   ```
+      ```bash
+      mkdir -p ~/ros2_ws/src
+      cd ~/ros2_ws/src/
+      ```
    - Clone the latest ROS2 Intel&reg; RealSense&trade;  wrapper from [here](https://github.com/IntelRealSense/realsense-ros.git) into '~/ros2_ws/src/'
-   ```bashrc
-   git clone https://github.com/IntelRealSense/realsense-ros.git -b ros2-beta
-   cd ~/ros2_ws
+      ```bashrc
+      git clone https://github.com/IntelRealSense/realsense-ros.git -b ros2-beta
+      cd ~/ros2_ws
+      ```
+### Step 4: Install dependencies
+   ```bash
+   sudo apt-get install python3-rosdep -y
+   sudo rosdep init # "sudo rosdep init --include-eol-distros" for Dashing
+   rosdep update
+   rosdep install -i --from-path src --rosdistro $ROS_DISTRO --skip-keys=librealsense2 -y
    ```
 
-  ### Step 4: Install dependencies:
+### Step 5: Build
    ```bash
-  sudo apt-get install python3-rosdep -y
-  sudo rosdep init # "sudo rosdep init --include-eol-distros" for Dashing
-  rosdep update
-  rosdep install -i --from-path src --rosdistro $ROS_DISTRO --skip-keys=librealsense2 -y
-  ```
-
-  ### Step 5: Build:
-  ```bash
-  colcon build
-  ```
+   colcon build
+   ```
 
-  ### Step 6: Source (on each new terminal):
-  ```bash
-  . install/local_setup.bash
-  ```
+### Step 6: Source (on each new terminal):

```

Comment:
These source commands set env variables to run ROS2 with the right distro.
You can add these source commands to your ~/.bashrc and then you will not need to do it on each terminal. 
**It's up to the user.**
I chose this way because I have multiple ROS2 distros on my machine, so I need to config each terminal with the right ROS2 distro setup